CONTRIBUTING.md

```diff
@@ -323,278 +317,6 @@ do not happen, so a parameter that accepts either `bytes` or
-at runtime. If the context manager never suppresses exceptions,
-have the return type be either `None` or `bool | None`. If you
-are not sure whether exceptions are suppressed or not or if the
-context manager is meant to be subclassed, pick `bool | None`.
-See https://github.com/python/mypy/issues/7214 for more details.
-
-`__enter__` methods and other methods that return instances of the
-current class should be annotated with `typing_extensions.Self`
-([example](https://github.com/python/typeshed/blob/3581846/stdlib/contextlib.pyi#L151)).
-
-### Naming
-
-Type variables and aliases you introduce purely for legibility reasons
-should be prefixed with an underscore to make it obvious to the reader
-they are not part of the stubbed API.
-
-A few guidelines for protocol names below. In cases that don't fall
-into any of those categories, use your best judgement.
-
-* Use plain names for protocols that represent a clear concept
-  (e.g. `Iterator`, `Container`).
-* Use `SupportsX` for protocols that provide callable methods (e.g.
-  `SupportsInt`, `SupportsRead`, `SupportsReadSeek`).
-* Use `HasX` for protocols that have readable and/or writable attributes
-  or getter/setter methods (e.g. `HasItems`, `HasFileno`).
-
-### `@deprecated`
-
-Typeshed uses the `@typing_extensions.deprecated` decorator
-(`@warnings.deprecated` since Python 3.13) to mark deprecated
-functionality; see [PEP 702](https://peps.python.org/pep-0702/).
-
-A few guidelines for how to use it:
-
-* In the standard library, apply the decorator only in Python versions
-  where an appropriate replacement for the deprecated functionality
-  exists. If in doubt, apply the decorator only on versions where the
-  functionality has been explicitly deprecated, either through runtime
-  warnings or in the documentation. Use `if sys.version_info` checks to
-  apply the decorator only to some versions.
-* Keep the deprecation message concise, but try to mention the projected
-  version when the functionality is to be removed, and a suggested
-  replacement.
-
-### Incomplete annotations
-
-When submitting new stubs, it is not necessary to annotate all arguments,
-return types, and fields. Such items should either be left unannotated or
-use `_typeshed.Incomplete` if this is not possible:
-
-```python
-from _typeshed import Incomplete
-
-field: Incomplete  # unannotated
-
-def foo(x): ...  # unannotated argument and return type
-```
-
-`Incomplete` can also be used for partially known types:
-
-```python
-def foo(x: Incomplete | None = None) -> list[Incomplete]: ...
-```
-
-### `Any` vs. `Incomplete`
-
-While `Incomplete` is a type alias of `Any`, they serve difference purposes:
-`Incomplete` is a placeholder where a proper type might be substituted.
-It's a "to do" item and should be replaced if possible. `Any` is used when
-it's not possible to accurately type an item using the current type system.
-It should be used sparingly.
-
-### "The `Any` trick"
```

Comment:
I think the `Any` trick is a good example of a "for more info, see also" kind of thing. Most contributors won't need to know about it, so it is IMO a good idea to link to it from CONTRIBUTING.md, but not explain it here.